infra/cifuzz/fuzz_target.py

```diff
@@ -121,6 +134,34 @@ def is_reproducible(self, test_case):
         return True
     return False
 
+  def download_latest_corpus(self):
+    """Downloads the latest OSS-Fuzz corpus for the target from google cloud.
+
+    Returns:
+      The local path to to corpus or None if download failed.
+    """
+    if not self.project_name:
+      return None
+    if not os.path.exists(self.out_dir):
+      logging.error('Out directory %s does not exist.', self.out_dir)
+      return None
+    corpus_dir = os.path.join(self.out_dir, 'backup_corpus', self.target_name)
+    os.makedirs(corpus_dir, exist_ok=True)
+    http_link = os.path.join(
```

Comment:
See my comments about url_join on the other PR (note that the [urljoin function](https://docs.python.org/3/library/urllib.parse.html#urllib.parse.urljoin) in the standard library doesn't work like we want).
The mixing of concatenation and join feels weird here.